Index: times
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Average assemble time = 0.268249\nAverage solver time = 0.0018208\nAverage RT Projection time = 0.138601\nAverage total time = 0.570209\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/times b/times
--- a/times	(revision 100d1ff9fe3331defe5470641b6639b4a88227c8)
+++ b/times	(date 1726261357189)
@@ -1,4 +1,4 @@
-Average assemble time = 0.268249
-Average solver time = 0.0018208
-Average RT Projection time = 0.138601
-Average total time = 0.570209
+Average assemble time = 0.0776647
+Average solver time = 0.00132971
+Average RT Projection time = 0.0565714
+Average total time = 0.192259
Index: Testing/Temporary/LastTest.log
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Start testing: Sep 10 13:06 CDT\n----------------------------------------------------------\nEnd testing: Sep 10 13:06 CDT\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Testing/Temporary/LastTest.log b/Testing/Temporary/LastTest.log
--- a/Testing/Temporary/LastTest.log	(revision 100d1ff9fe3331defe5470641b6639b4a88227c8)
+++ b/Testing/Temporary/LastTest.log	(date 1726261353809)
@@ -1,3 +1,3 @@
-Start testing: Sep 10 13:06 CDT
+Start testing: Sep 13 16:02 CDT
 ----------------------------------------------------------
-End testing: Sep 10 13:06 CDT
+End testing: Sep 13 16:02 CDT
